owncloudData/src/main/java/com/owncloud/android/data/sharing/shares/datasources/mapper/RemoteShareMapper.kt

```diff
@@ -61,15 +55,11 @@ class RemoteShareMapper : RemoteMapper<OCShare, RemoteShare> {
                 sharedWithAdditionalInfo = model.sharedWithAdditionalInfo!!,
                 name = model.name!!,
                 shareLink = model.shareLink!!,
-                fileSource = model.fileSource,
-                itemSource = model.itemSource,
-                shareType = RemoteShareType.fromValue(model.shareType.value),
+                shareType = com.owncloud.android.lib.resources.shares.ShareType.fromValue(model.shareType.value),
```

Comment:
Can't you use `import com.owncloud.android.lib.resources.shares.` and directly use `ShareType.fromValue(model.shareType.value)` ?

Reply:
No, because it has name collision with the other sharetype. But i can use an alias to import it as it was done before and call it RemoteShareType. 👍